.github/workflows/release-docker.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: Build CI Base Image
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - docker/ci-base/**
+      - go.mod
+      - go.sum
+      - .taskfiles/install/Taskfile.yml
+
+jobs:
+  base-ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Tyk
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+
+      - uses: docker/setup-buildx-action@v2
+
+      - run: cp go.mod go.sum .taskfiles/install/Taskfile.yml docker/ci-base/data
+
+      - name: CI base image
+        uses: docker/build-push-action@v3
+        with:
+          push: false
+          context: .
+          file: docker/ci-base/Dockerfile
+          platforms: linux/amd64
+          build-args: |
+            GITHUB_SHA=${{ github.sha }}
+            GITHUB_TAG=${{ github.ref }}
+          tags: |
+            tykio/base-ci:5.1
```

Comment:
@buger @alephnull could we use this image tag, or should we use something else, tyklabs/base-ci or something?

Reply:
@buger @alephnull we should connect in regards to this, leon wants ECR, i want a vanity domain on it like `registry.tyk.io`, something we can publish without `aws.com` or some random hash in the name...